src/CitMovie.Api/Controller/PromotionalMediaController.cs

```diff
@@ -0,0 +1,122 @@
+using System.Security.Cryptography.X509Certificates;
+
+namespace CitMovie.Api;
+
+[ApiController]
+[Route("api/promotional_media")]
```

Comment:
The endpoint should be `api/media/{mediaId}/releases/{releaseId}/promotional_media`